res/controllers/MIDI_for_LightEffects.midi.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    MIDI for light effects
+
+    send midi information about the musing to generate light effects with a light control app like QLCplus
+
+    This file should be placed into /usr/share/mixxx/controllers
+-->
+<MixxxMIDIPreset mixxxVersion="1.10+" schemaVersion="1">
+	<info>
+		<name>MIDI for LightEffects</name>
+		<author>DG3NEC / Michael Stahl - Markus Baertschi</author>
+		<description>MIDI-Output for light effects</description>
+		<forums>http://www.mixxx.org/forums</forums>
```

Comment:
This is usually a forum post where users can go for support. Can you make a post in the controllers sub-forum and then put the link here? That way users have a place to go to ask questions.

http://mixxx.org/forums/viewforum.php?f=7&sid=fc57cf579ddfad0bda44bc5125b819ad